tensorflow/lite/experimental/micro/examples/micro_speech/apollo3/captured_data_to_wav.py

```diff
@@ -0,0 +1,40 @@
+# SPDX-License-Identifier: Apache-2.0
```

Comment:
I can't add a comment to apollo3.h since it's too large, but can we add it to a separate repo?